web_list_html_widget/README.rst

```diff
@@ -0,0 +1,45 @@
+Widget Html rendering in list views
+===================================
+
+This module allows to use a ``widget="html"`` for fields in list(tree) views.
+
+*WARNING* The content of the field will be rendered without HTML escaping and without any other security measure.
+
+Features
+========
+
+* Add widget 'html' in a Html field on a tree view
+
+
```

Comment:
Missing the try on runbot somewhere here, have a look at the template:

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst